pkg/cmd/server/start/node_args.go

```diff
@@ -114,6 +114,11 @@ func (args NodeArgs) BuildSerializeableNodeConfig() (*configapi.NodeConfig, erro
 		DNSIP:     dnsIP,
 
 		MasterKubeConfig: admin.DefaultNodeKubeConfigFile(args.CertArgs.CertDir, args.NodeName),
+
+		PodManifestConfig: &configapi.PodManifestConfig{
```

Comment:
Either is fine, same end result. Explicitly setting to nil makes it clear it was intentionally omitted by default